### PR TITLE
Add explorations deployments to mainnet config

### PR DESCRIPTION
**Pull request by ryley-o**
Add Art Blocks Explorations deployments to mainnet config file.

Adds deployed contracts as defined in: https://github.com/ArtBlocks/artblocks-contracts/pull/349

Note that only the Merkle Minter V1 was added in this deployment.

### CODE REVIEW

**Review by jakerockland — APPROVED** [review]

**Comment by ryley-o** [comment]
note: this branch has been deployed to studio, and is currently syncing (not yet published). release draft here: https://github.com/ArtBlocks/artblocks-subgraph/releases/edit/untagged-e3f3a2a183a4d1051cd1

**Comment by ryley-o** [comment]
Also note that this branch has been deployed to hosted via grafting. Release documented here: https://github.com/ArtBlocks/artblocks-subgraph/releases/tag/v0.0.21-hosted%2Fmainnet

**Comment by jakerockland** [comment]
@ryley-o I've added this commit https://github.com/ArtBlocks/artblocks-subgraph/pull/128/commits/dc8f75d2420681f3eb208a87735daa161b764f90 to this PR to account for this change over in the contracts repo: https://github.com/ArtBlocks/artblocks-contracts/pull/351

File: config/mainnet.json [hunk 1]
```diff
@@ -233,13 +233,18 @@
...
     {
       "address": "0x39F0e1e8Df2C135Df35B3FA496e943ac4B90Fe0f",
-      "name": "Admin ACL V0 for V3 core",
+      "name": "(deprecated) Admin ACL V0 for V3 core",
       "startBlock": 15726705
     },
     {
       "address": "0x286bA5808A98c9237449182E05740E4B7284389e",
-      "name": "Admin ACL V0 for Explorations",
+      "name": "(deprecates) Admin ACL V0 for Explorations",
```
Comment (jakerockland) [inline config/mainnet.json:241]: Spelling nit s/deprecates/deprecated

**Comment by ryley-o** [comment]
@jakerockland - I've added commit [ede98a8](https://github.com/ArtBlocks/artblocks-subgraph/pull/128/commits/ede98a82bc0d5ba42a927949c72778037e6d9680) with the newly deployed AdminACLV1 contract to our mainnet config. For subgraph indexing purposes, it may be treated as an AdminACLV0 at this time (since we are not indexing the set of addresses allowed to approve artist payment address updates)

File: config/mainnet.json (hunk 1 above)
Comment (ryley-o) [reply config/mainnet.json]: 🙏

File: config/mainnet.json [hunk 2]
```diff
@@ -233,13 +233,18 @@
...
       "startBlock": 15791055
+    },
+    {
+      "address": "0x286bA5808A98c9237449182E05740E4B7284389e",
```
Comment (jakerockland) [inline config/mainnet.json:245]: This should be `0x18b18cF97a3D8BCC43f8D1Df282CebA85f717C82` IIUC.

Comment (ryley-o) [reply config/mainnet.json]: oof - good catch 😮‍💨 🥇 fixed!

Comment (jakerockland) [reply config/mainnet.json]: huzzah! 🙏 :vibe:

**Comment by ryley-o** [comment]
Deployed via grafting to hosted mainnet subgraph, as documented in this release: https://github.com/ArtBlocks/artblocks-subgraph/releases/tag/v0.0.22-hosted%2Fmainnet

**Comment by ryley-o** [comment]
Note that the latest commit has been deployed to the decentralized network at v0.0.22, as documented in this release: https://github.com/ArtBlocks/artblocks-subgraph/releases/tag/v0.0.22-decentralized%2Fmainnet

**Comment by ryley-o** [comment]
@lyaunzbe - gentle ping on this one, since it is deployed everywhere I think an easy ✅  for you

**Pull request merged**